aim/mixins.py

```diff
@@ -27,9 +27,9 @@ def forward(
         x, feats = self.trunk(x, mask=mask, max_block_id=max_block_id)
         output["trunk_output"] = feats
 
-        x = self.head(x, mask=mask)
+        logits, descriptor = self.head(x, mask=mask)
 
-        return x, output
+        return logits, descriptor
```

Comment:
I guess keeping the `output` variable is now useless and can be removed.
The return type should then be changed to `Tuple[ArrayLike, ArrayLike]`.